api/v1beta2/cluster_types.go

```diff
@@ -585,6 +585,15 @@ type Topology struct {
 	// +listMapKey=name
 	// +kubebuilder:validation:MaxItems=1000
 	Variables []ClusterVariable `json:"variables,omitempty"`
+
+	// nodeDeletionStrategy specifies the strategy to delete nodes in the cluster.
+	// Available options:
+	// - graceful
+	// - force
+	// By default it's "force".
+	// +optional
+	// +kubebuilder:validation:Enum=force;graceful
+	NodeDeletionStrategy *NodeDeletionStrategyType `json:"nodeDeletionStrategy,omitempty"`
```

Comment:
This doesn't need to be a pointer. The zero value isn't a valid choice and `omitempty` is already present.